History.md

```diff
@@ -1,3 +1,7 @@
+## v2.0.1
+
+* Wrap localStorage in try/catch to avoid crash when disabled. Port fix from v1 branch. #182
```

Comment:
I think `#182` should be a link here. So wrap it into a link.

Reply:
Fixed